cryptoinvestor/models/user.py

```diff
@@ -9,64 +9,63 @@ def __init__(self, *, username: str, password: str, account: Account):
         self.username = username
         self.password = password
         self.account = account
-        self.bought = dict()
 
-    def buy(self, count: float, a_name: str) -> float:
+    def buy(self, amount: float, a_name: str) -> float:
         asset = self.account.assets.get(a_name)
         if not asset or not asset.is_loaded():
             raise self.Error(f"{a_name} is not a valid cryptocurrency or is not loaded")
 
-        if count < 0:
-            raise self.Error(f"{count} is not a valid number of units")
+        if amount < 0:
+            raise self.Error(f"{amount} is not a valid number of units")
 
         lc = self.account.local_currency
 
         price = asset.rates.get(lc).get('rate')
 
-        s = count * price
+        s = amount * price
 
         if self.balance() < s:
             raise self.Error(
-                f"Buying {count} of {a_name} would cost {s} {lc}. You have {self.balance()} {lc}"
+                f"Buying {amount} of {a_name} would cost {s} {lc}."
+                f"You have {self.balance()} {lc}"
             )
 
-        owned = self.bought.get(a_name, 0)
-
-        self.bought[a_name] = owned + count
-        self._buy(s)
+        print(self.account.investments)
```

Comment:
If you need to log something use `logging` otherwise please remove print statements